tests/std/include/range_algorithm_support.hpp

```diff
@@ -353,6 +354,10 @@ namespace test {
             return *this;
         }
 
+        constexpr operator iterator<Category, const Element, Diff, Eq, Proxy, Wrapped>() && noexcept {

```

Comment:
Why not use the `Consterator` typedef here?

Reply:
```suggestion
        constexpr operator Consterator() && noexcept {
```